example.js

```diff
@@ -70,6 +71,24 @@ const TermsOfStorage = () => (
   </>
 )
 
+// Define custom map overlays.
+// customMapOverlays can be a single overlay element or an array of such elements.
+// Each overlay must include a name prop (and a key prop if wrapping in an array).
+// (Wrapping the overlays inside a React Fragment <> or other component will not work.)
+const customMapOverlays = [
+  <GtfsRtVehicleOverlay
+    key='custom1'
+    liveFeedUrl='https://gtfs-rt.example.com/feed1.pb'
+    name='GTFS-rt Example Vehicles 1'
+  />,
+  <GtfsRtVehicleOverlay
+    key='custom2'
+    liveFeedUrl='https://gtfs-rt.example.com/feed2.pb'
+    name='GTFS-rt Example Vehicles 2'
+    routeDefinitionUrl='https://gtfs-rt.example.com/routes.json'
+  />
+]
+
```

Comment:
I'm not sure we should be including these non-functional overlays in the `example.js` file. Maybe we can just comment out the contents (and just pass an empty array).

Reply:
Makes sense, updated in https://github.com/opentripplanner/otp-react-redux/pull/401/commits/4348cdb2c20d2658d9b4f113dd15e18d3b110939.